groups/utils.js

```diff
@@ -104,6 +104,29 @@ async function addGroupRoleToMember(memberRoleBody) {
   }
 }
 
+async function removeRoleFromMember(roleId, discordId) {
```

Comment:
what if the HTTP response is not successful, but it does not specify what kind of error occurred. This can make it difficult to debug problems with the function.

Reply:
res! removed and console.error added